scripts/rebuild_languages.py

```diff
@@ -33,104 +35,120 @@ def downloadSheet(url):
+
+    languageSourceTemplate = """\
+    // Don't edit this file: it is generated by scripts/rebuild_languages.py
+    // The strings are maintained here: {url}
+    export const strings = {{
+    {stringsBlock}
+    }};
+    """
+
+    for languageCode, strings in languageStrings.items():
+        languagePath = langDir / f"{languageCode.strip()}.js"
+        print("writing", languagePath)
+        lines = []
+        for k, v in sorted(strings.items()):
+            lines.append(f"  {jsonDump(k.strip())}: {jsonDump(v)},")
+        stringsBlock = "\n".join(lines)
+        languageSource = languageSourceTemplate.format(
+            stringsBlock=stringsBlock, url=languageSpreadsheetURL
+        )
+        languagePath.write_text(languageSource, encoding="utf-8")
+        prettier(languagePath, repoDir)
 
-    for row in rows:
-        key = row[1]
-        if not key.strip():
-            continue
-        string = row[columnIndex]
-        if not string or string == "-":
-            string = languageStrings["en"].get(key, "!missing!")
-        strings[key] = string
-
-
-repoDir = pathlib.Path(__file__).resolve().parent.parent
-langDir = repoDir / "src" / "fontra" / "client" / "lang"
-assert langDir.is_dir()
-localizationJSPath = repoDir / "src" / "fontra" / "client" / "core" / "localization.js"
-assert localizationJSPath.is_file()
-
-localizationJSSource = localizationJSPath.read_text(encoding="utf-8")
-
-languagesList = "\n".join(f"  {jsonDump(langs)}," for langs in languages)
-languagesBlock = f"""// Don't edit this block, see scripts/rebuild_languages.py
-export const languages = [
-{languagesList}
-];
-"""
-
-indexStart = localizationJSSource.find(languagesBlock.splitlines()[0])
-assert indexStart > 0
-indexEnd = localizationJSSource.find("];\n", indexStart)
-assert indexEnd > indexStart
-
-localizationJSSource = (
-    localizationJSSource[:indexStart]
-    + languagesBlock
-    + localizationJSSource[indexEnd + 3 :]
-)
-
-localizationJSPath.write_text(localizationJSSource, encoding="utf-8")
-prettier(localizationJSPath)
-
-
-languageSourceTemplate = """\
-// Don't edit this file: it is generated by scripts/rebuild_languages.py
-// The strings are maintained here: {url}
-export const strings = {{
-{stringsBlock}
-}};
-"""
-
-for languageCode, strings in languageStrings.items():
-    languagePath = langDir / f"{languageCode.strip()}.js"
-    print("writing", languagePath)
-    lines = []
-    for k, v in sorted(strings.items()):
-        lines.append(f"  {jsonDump(k.strip())}: {jsonDump(v)},")
-    stringsBlock = "\n".join(lines)
-    languageSource = languageSourceTemplate.format(
-        stringsBlock=stringsBlock, url=languageSpreadsheetURL
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description="Script to rebuild the language files from the Google Sheet"
+    )
+    parser.add_argument(
+        "--wip", action="store_true", required=False, help="Flag include WIP rows"
```

Comment:
I think you can omit `required=False`, 99% sure that's the default.

Please make the help text say:
"Include work-in-progress strings, as marked with 'WIP' in the the first column."

Reply:
done.